.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint
```

Comment:
these actions dont appear to be triggering yet, maybe they need to be set up on repo side too (but i wouldnt have permissions).

Reply:
instead of PR'ing within your own `andyborn:hk/modernise` fork, you should be (now) able to PR directly in this repo